source/extensions/filters/network/mysql_proxy/mysql_codec.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+#include <cstdint>
+
+#include "envoy/common/platform.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/common/logger.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace MySQLProxy {
+
+constexpr int MYSQL_MAX_STR_SIZE = 256;
```

Comment:
prefer specific types everywhere. I see a lot of usages of `int` in this change. Can you do a full audit and switch to an appropriate probably unsigned type?